packages/lumx-react/src/index.ts

```diff
@@ -48,6 +48,8 @@ export { ListItem, ListItemProps, ListItemSize, ListItemSizes } from './componen
 
 export { ListSubheader, ListSubheaderProps } from './components/list/ListSubheader';
 
+export { CLASSNAME, DEFAULT_PROPS, MessageKind, Message, MessageProps } from './components/message/Message';
```

Comment:
I think the `CLASSNAME` and `DEFAULT_PROPS` don't need to be exported here (at least we didn't find reasons to do it for other components).
If we need them to be exported, we would all need to rename them I think

Reply:
I export them for use in my tests. Do I have to remove exports and duplicate new `DEFAULT_PROPS` and `CLASSNAME` for tests?

Reply:
The tests can use these from the `components/message/Message.tsx` file directly rather than in this `index.ts` file